net.lutris.Lutris.yml

```diff
@@ -33,17 +33,17 @@ finish-args:
 add-extensions:
   org.freedesktop.Platform.Compat.i386:
     directory: lib/i386-linux-gnu
-    version: "18.08"
+    version: "19.08beta"
 
   org.freedesktop.Platform.Compat.i386.Debug:
     directory: lib/debug/lib/i386-linux-gnu
-    version: "18.08"
+    version: "19.08beta"
     no-autodownload: true
 
   org.freedesktop.Platform.GL32:
     directory: lib/i386-linux-gnu/GL
     version: "1.4"
-    versions: "18.08;1.4"
+    versions: "19.08beta;1.4"
```

Comment:
1.4 is still needed?

Reply:
Yes, it's the default branch for current Nvidia GL extensions.